apps/emqx/src/emqx_schema.erl

```diff
@@ -2285,6 +2285,16 @@ common_ssl_opts_schema(Defaults) ->
                     desc => ?DESC(common_ssl_opts_schema_secure_renegotiate)
                 }
             )},
+        {"log_level",
+            sc(
+                hoconsc:enum([
+                    emergency, alert, critical, error, warning, notice, info, debug, none, all
+                ]),
+                #{
+                    default => notice,
+                    desc => ?DESC(common_ssl_opts_schema_log_level)
```

Comment:
Maybe mark log_level as `importance => ?IMPORTANCE_LOW` is better ？ 
The number of users who need to modify this parameter should be small.

Reply:
OK